packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-sign-in-pwd-phone/main/nodejs/integrationsteps.md

```diff
@@ -24,14 +34,24 @@ After this response, you need to redirect the user to an authenticator list page
 
 ### 2: The user selects the phone factor from the authenticator list
 
-In this use case, only the **Phone** factor appears in the list of authenticators, as shown in the following example page:
+In this use case, only the **Phone** factor appears in the list of authenticators, similar to the following wireframe:
 
-<div class="half border">
+<div class="half wireframe-border">
 
-![Displays a Select Authenticator page that includes a phone option and a Select button.](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-sign-in-pwd-phone-screen-verify-nodejs.png)
+![Choose authenticator form with phone option](/img/oie-embedded-sdk/wireframes/choose-authenticator-phone-form-g3r18.png)
```

Comment:
Title should be "Choose an authenticator" rather than "Choose your authenticator"